docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -13,271 +13,245 @@ it and join the cluster.
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- AWS account with EC2 instances and permissions to create and attach IAM policies as well as permission to create Systems Manager documents.
-- EC2 instances running Ubuntu/Debian/RHEL/Amazon Linux 2 if making use of the default Teleport install script. (For other Linux distributions, you can install Teleport manually.) 
+- AWS account with EC2 instances and permissions to create and attach IAM
+policies as well as permission to create Systems Manager documents.
```

Comment:
```suggestion
policies.
```

I'm not sure if the user's AWS account also needs this permission any more with the new bootstrap command. Defer to @lxea.